autoprocessing/autoprocess.py

```diff
@@ -1,85 +0,0 @@
-import os
```

Comment:
I think we should add a functionallity like this for bootstrax. Since right now we use the pc to do everything. It would be nice to have the option to submit jobs to stoomboot for the processing rather than doing everything locally

Reply:
In case bootstrax is not able to keep up or we overload the machine, this is true. But it should be fairly trivial to add since we write to stoomboot directories directly.

We can always go back in the git history and easily refurbish this script.